Donation-platform/src/main/java/com/google/servlets/PersonalizationServlet.java

```diff
@@ -38,30 +39,24 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Parse tags JSON from request into a list of tags
```

Comment:
Add a comment on the format of requestData that you are trying to parse

Reply:
+1

Reply:
Add example

Reply:
Added an example of requestData.